internal/configconverter/config_server_test.go

```diff
@@ -34,6 +35,15 @@ import (
 
 func TestConfigServer_RequireEnvVar(t *testing.T) {
 	waitForRequiredPort(t, defaultConfigServerPort)
+	url := fmt.Sprintf("http://%s/debug/configz/initial", defaultConfigServerEndpoint)
+	client := &http.Client{}
+
+	require.EventuallyWithT(t, func(c *assert.CollectT) {
+		assert.True(c, isPortAvailable(defaultConfigServerPort))
```

Comment:
This is probably unnecessary, as the check at the beginning of the method, `waitForRequiredPort` is a `required.EventuallyWithT` call to this method waiting for it to be available. With the flakiness we've been seeing though, I assume it can't hurt.